src/Admin/AmpPlugins.php

```diff
@@ -291,17 +291,19 @@ public function filter_plugins_api( $response, /** @noinspection PhpUnusedParame
 	public function filter_action_links( $actions, $plugin ) {
 
 		if ( isset( $plugin['wporg'] ) && true !== $plugin['wporg'] ) {
-			$actions = [];
+			$actions       = [];
+			$external_icon = '<span aria-hidden="true" class="dashicons dashicons-external"></span>';
 
 			if ( ! empty( $plugin['homepage'] ) ) {
 				$actions[] = sprintf(
-					'<a href="%s" target="_blank" rel="noopener noreferrer" aria-label="%s">%s</a>',
+					'<a href="%s" target="_blank" rel="noopener noreferrer" aria-label="%s">%s<span class="screen-reader-text">(opens in a new tab)</span>%s</a>',
```

Comment:
I just realized this is missing translation. See https://github.com/ampproject/amp-wp/pull/6723.